src/Controls.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component, Fragment } from 'react';
+import './controls.css';
+
+class Controls extends Component {
+  render() {
+    return (
+      <Fragment>
+        <div className="controls">
+          <div className="controls-title">Instructions to Play</div>
+          <ol>
+
+            <li>1. Press <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="36" height="36">
```

Comment:
Use `ul` instead of `ol` and remove the 1, 2, 3 you have yourself added. If you did this because of weird indentation of line items. See if this fixes it https://github.com/itaditya/trick-or-treat-game/blob/master/src/scoreboard.css#L22